### ISSUE TITLE
No Tampermonkey Permission Request

### ISSUE THREAD

**Issue by floriankilian** [hoblin/x-tracker#49]
**Describe the Bug**

**Steps to Reproduce**
Installing a new X-Tracker Tampermonkey script (in my case #16). Neither during install nor the first visit on the tweets page there was a prompt requesting permission to send data to X-Tracker.

**Expected Behavior**
"A prompt will appear, asking for your permission to send data to X-Tracker. Click "Allow" to start tracking the tweet metrics." ([link](https://x-tracker.hobl.in/tweets/new))

**Environment (please complete the following information):**
- OS: Windows 11
- Browser Chrome
- Version 17.0.5938.92

**Additional Context**
The Tampermonkey script runs (as it reloads the tweets page), however there is also no data being displayed on the [tracked tweet](https://x-tracker.hobl.in/tweets/16). So not sure if the prompt requesting was not executed by Tampermonkey or a link is missing in the user script?

**Comment by floriankilian** [comment]
With your newest update it shows the three tweets as "never tracked" (16/17/18)

Also just saw this warning in the Tampermonkey script within the Tampermonkey plugin:
<img width="316" alt="image" src="https://github.com/hoblin/x-tracker/assets/34104015/ed7da5a4-2630-4562-a4ff-f280fa7ccec5">

eslint: no-undef - 'jQuery'is not defined.

**Comment by floriankilian** [comment]
-> just changed my language settings to english, and now it prompted me for permission and it's tracking! :)

<img width="572" alt="image" src="https://github.com/hoblin/x-tracker/assets/34104015/704254a9-c293-462c-9282-161f747781cd">

**Comment by hoblin** [comment]
Right, the script looks for the `aria-label` attributes and expects English text. Good catch. Thanks!

**Comment by hoblin** [comment]
We can add an object with collection of strings and then detect page locale

```js
const preset = {
en: {like: 'like', // etc
```
